jobs/ci-kubernetes-e2e-gke-container_vm-1.3-container_vm-latest-upgrade-cluster-new.env

```diff
@@ -0,0 +1,13 @@
+### job-env
+
+E2E_OPT=--check_version_skew=false
+E2E_UPGRADE_TEST=true
+GINKGO_UPGRADE_TEST_ARGS=--ginkgo.focus=\[Feature:ClusterUpgrade\] --upgrade-target=ci/latest-latest --upgrade-image=container_vm
+JENKINS_PUBLISHED_SKEW_VERSION=ci/latest-latest
```

Comment:
Everyone hates bash... ( you always need to look it up ) that's why we want to move away from it! :-p